### ISSUE TITLE
idea plugin instance not starting with ["org.intellij.scala:2022.2.17".toPlugin, intellijBuild := "222.3739.54", scalaVersion := "2.13.8"]

### ISSUE THREAD

**Issue by moglideveloper** [JetBrains/sbt-idea-plugin#119]
intellij plugin not getting starting with below configuration :-

```
scalaVersion := "2.13.8"
ThisBuild / intellijBuild := "222.3739.54"
intellijPlugins += "org.intellij.scala:2022.2.17".toPlugin
```

getting below error :-

```
2022-11-05 21:00:56,836 [     29]   WARN - #c.i.i.StartupUtil - Unable to load JNA library (OS: Mac OS X 11.6.5)
java.lang.UnsatisfiedLinkError: Could not find JNA native support
```

Full logs : https://pastebin.com/Ahi9qEjE

Very simplified plugin to reproduce this issue : https://github.com/moglideveloper/SamplePlugin

**Comment by unkarjedy** [comment]
Which version of sbt-idea-plugin you use?
Did you try updating to the latest `3.17.9`?

**Comment by techleadAtCodogenic** [comment]
`addSbtPlugin("org.jetbrains" % "sbt-idea-plugin" % "3.18.0")`


https://github.com/moglideveloper/SamplePlugin/blob/master/project/plugins.sbt

Also, attached logs https://pastebin.com/Ahi9qEjE has one more exception :-

`Caused by: java.lang.NoClassDefFoundError: Could not initialize class com.intellij.ui.mac.foundation.Foundation`


Almost similar errors are reported in below reports :-

https://youtrack.jetbrains.com/issue/IDEA-265943/Due-to-JNA-outdate-runIde-failed-on-Apple-silicon-with-IC-202033

https://youtrack.jetbrains.com/issue/IDEA-261478/Build-of-latest-stable-intellij-community-fails-on-Apple-Silicon

https://youtrack.jetbrains.com/issue/IDEA-260376?_ga=2.44141860.1209135726.1667963123-167516747.1644845016&_gl=1*1c2x8to*_ga*MTY3NTE2NzQ3LjE2NDQ4NDUwMTY.*_ga_9J976DJZ68*MTY2Nzk2MzEyMy40OC4xLjE2Njc5NjMxNDkuMC4wLjA.

https://intellij-support.jetbrains.com/hc/en-us/community/posts/360010581159-Can-t-build-plugin-Fatal-error-initializing-com-intellij-ide-ui-LafManager-

**Comment by moglideveloper** [comment]
addSbtPlugin("org.jetbrains" % "sbt-idea-plugin" % "3.18.0")

https://github.com/moglideveloper/SamplePlugin/blob/master/project/plugins.sbt

Also, attached logs https://pastebin.com/Ahi9qEjE has one more exception :-

Caused by: java.lang.NoClassDefFoundError: Could not initialize class com.intellij.ui.mac.foundation.Foundation

Almost similar errors are reported in below reports :-

https://youtrack.jetbrains.com/issue/IDEA-265943/Due-to-JNA-outdate-runIde-failed-on-Apple-silicon-with-IC-202033

https://youtrack.jetbrains.com/issue/IDEA-261478/Build-of-latest-stable-intellij-community-fails-on-Apple-Silicon

https://youtrack.jetbrains.com/issue/IDEA-260376?_ga=2.44141860.1209135726.1667963123-167516747.1644845016&_gl=1*1c2x8to*_ga*MTY3NTE2NzQ3LjE2NDQ4NDUwMTY.*_ga_9J976DJZ68*MTY2Nzk2MzEyMy40OC4xLjE2Njc5NjMxNDkuMC4wLjA.

https://intellij-support.jetbrains.com/hc/en-us/community/posts/360010581159-Can-t-build-plugin-Fatal-error-initializing-com-intellij-ide-ui-LafManager-

**Comment by unkarjedy** [comment]
The issue should be fixed in version 3.18.2
(I should be published in 2-3 hours)
